policies/strategic_planning_committee_draft.md

```diff
@@ -0,0 +1,53 @@
+Are you interested to join Astropy Strategic Planning Committee?
+Here is what to expect.
+
+## Time commitment
+
+* Meet at least monthly. Each meeting could be 30-60 minutes long.
+  Notes should be publicly available.
+* Must be willing to attend the annual Astropy Coordination Meeting, either
+  in-person or remotely. This is about a week. Or if you join remotely just
+  for certain sessions, then a few days.
+* Outside of meetings, must be willing to invest time in actually accomplishing
+  duties laid out below. Time estimate is tough here because it depends on
+  workload at a given time, and how that load is divided among committee members.
+  Maybe no more than a few days a month?

```

Comment:
For a reasonably sized committee I suspect it's less than a few days a month.  More like a few days a year?  Spread out in hour-long chunks of course, but still.

Reply:
Can people really do strategic planning for worldwide project by only investing a few days a year worth of work? 🤔

Reply:
I am hesitant to state "a few days a year" at this point, so I will not accept this suggestion for this round.